ansible/roles/common/tasks/main.yml

```
@@ -39,6 +52,16 @@
   retries: 10
   delay: 15
   until: yumresult is not failed
+- name: "enable SELinux (Red Hat)"
+  block:
+    - name: "check if container-selinux is installed"
+      shell: "rpm -q --quiet container-selinux"
+      register: container_selinux_installed
+      failed_when: container_selinux_installed.rc == 2
+    - name: "install container-selinux"
+      shell: "sudo yum -y install http://mirror.centos.org/centos/7/extras/x86_64/Packages/container-selinux-2.107-1.el7_6.noarch.rpm"
```

Could the link use `https` instead of `http`?  Also could add a comment like the following.

```suggestion
      # This makes docker work on RHEL 7
      shell: "sudo yum -y install http://mirror.centos.org/centos/7/extras/x86_64/Packages/container-selinux-2.107-1.el7_6.noarch.rpm"
```

---

I updated the comment. There isn't an HTTPS version of the default repo for CentOS. I could use a mirror that supports HTTPS, but I haven't found an elegant way to find the best mirror like the method in `get-asf-mirror.yml`. CentOS does have a status of mirrors at https://mirror-status.centos.org/ but it only lists one HTTPS mirror. I am open to suggestions if and how I should approach this.

---

If using RHEL7, it seems better to enable the extras subscription rather than pull this RPM from the CentOS repos at all. From https://fedoraproject.org/wiki/EPEL :

```
on RHEL 7 it is recommended to also enable the optional, extras, and HA repositories since EPEL packages may depend on packages from these repositories:
   # subscription-manager repos --enable "rhel-*-optional-rpms" --enable "rhel-*-extras-rpms"  --enable "rhel-ha-for-rhel-*-server-rpms"
```

---

Enabling these repositories require an active subscription to RedHat. I was hoping that wasn't necessary and a user could just use a RHEL AMI.

---

I'm not sure it makes sense to cater to a RHEL AMI without a subscription. Why wouldn't such a user just use a CentOS AMI? It doesn't seem worth the added complexity, and I'm having difficulty imagining that this satisfies a need within the Fluo/Accumulo developer community.

As Muchos is not a released product of the Fluo PMC, and is instead an internal tool for testing Fluo/Accumulo themselves, I'm finding it difficult to justify supporting this hybrid RHEL/CentOS approach.

---

The environment that I am working on has restrictions on certain AMIs. For one reason or another, CentOS is not an image that can be used, but RHEL is an image that can be used. I originally tried to adapt Fluo for Amazon Linux, but found it much easier to just adapt for RHEL.

---

Eek, that environment doesn't seem very developer-friendly. Still, I wonder if that's a general use case we want to add complexity to support for Fluo/Accumulo developers at large, or if it's a "this is just for me" kind of use case that makes sense in only your fork. (I have a similar "just for me" fork of fluo-uno for my use cases.)

---

Yeah, not the best env to get things done.

It is probably best for me to just keep this as a branch on a fork. Thanks!